<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SqlQueryDialog</class>
 <widget class="QWidget" name="SqlQueryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Noto Sans</family>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>SQL-запрос</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/Resources/Logo.jpg</normaloff>:/Resources/Logo.jpg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelInputQuery">
         <property name="font">
          <font>
           <family>Noto Sans</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Введите запрос:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="buttonClear">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Resources.qrc">
             <normaloff>:/Resources/Trash.png</normaloff>:/Resources/Trash.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonRun">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Resources.qrc">
             <normaloff>:/Resources/Run.png</normaloff>:/Resources/Run.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="textQuery">
       <property name="font">
        <font>
         <family>Noto Sans</family>
         <pointsize>16</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="labelResult">
       <property name="font">
        <font>
         <family>Noto Sans</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Результат:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textResult">
       <property name="font">
        <font>
         <family>Noto Sans</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
